distribution/docs/src/main/jdocs/content/_plugins/default-security-attribute-plugin-contents.adoc

```diff
@@ -0,0 +1,34 @@
+:type: plugin
+:status: published
+:title: Default Security Attribute Plugin
+:link: _default_security_attribute_plugin
+:plugintypes: preingest
+:summary: Maps user attributes to metacard attributes for system user.
```

Comment:
`for system user` doesn't sound right to me. `from the system user` might be more accurate, since I believe this occurs regardless of the user performing the operation? Same phrasing on the line below too.